### DISCUSSION TITLE
Where can I learn about typst-show.typ?

### DISCUSSION THREAD

**Discussion by fredguth** [quarto-dev/quarto-cli#10449] [Q&A]
### Description

I am trying to create a Typst format extension and I don't understand `_extensions/my_extension/typst-show.typ`

What is it? How does it work?

I have been using it in a copy and paste fashion so far, every time I create a new yaml entry, but in trying to implement some advanced features (see #10446), I believe my lack of understanding is limiting me.

**Comment by mcanouil** [comment]
See <https://quarto.org/docs/journals/templates.html#typst-partials>.

Also see Pandoc templating framework: https://pandoc.org/MANUAL.html#template-syntax
For everything Typst, see <https://typst.app/docs/>.
